web/src/features/moreCast2/components/GridComponentRenderer.tsx

```diff
@@ -120,7 +121,16 @@ export class GridComponentRenderer {
     // The grass curing 'forecast' field is rendered differently
     if (isGrassField) {
       return (
-        <ValidatedForecastCell
+        <ValidatedGrassCureForecastCell
+          disabled={isActual || isPreviousDate}
+          label={label}
+          value={params.formattedValue}
+          validator={validator}
+        />
+      )
+    } else if (field.includes('windDirection')) {
+      return (
+        <WindDirectionForecastCell
```

Comment:
Unsure about why it renders wide if it's not split out, would need to play around with it for awhile. That was happening when it was just being rendered as a ValidatedForecastCell?

nit: maybe stick with the 'Validated' prefix for the Cell component name? Also might be good to put a note in here about why WindDirectionForecastCell is split out

Reply:
> That was happening when it was just being rendered as a ValidatedForecastCell?

Yep. I'll rename with the Validated prefix too, thanks.

Reply:
Hmm, maybe it wasn't getting rendered as a ValidatedForecastCell because it only renders that way if it `isGrassField`. No need to respond if that's way off, Morecast hurts my brain sometimes